<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>JanelaPrincipal</class>
 <widget class="QMainWindow" name="JanelaPrincipal">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>660</width>
    <height>374</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Calcular</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>N:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="valorN"/>
      </item>
      <item>
       <widget class="Line" name="line">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>K:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="valorK"/>
      </item>
      <item>
       <widget class="Line" name="line_2">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Soma: </string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="valorSoma"/>
      </item>
      <item>
       <widget class="Line" name="line_3">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Nº Soluções:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="numSolucoes"/>
      </item>
      <item>
       <widget class="QPushButton" name="Calcular">
        <property name="text">
         <string>Calcular</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QListView" name="resultadosList"/>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="Salvar">
        <property name="text">
         <string>Salvar Arquivo</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>660</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_Ajuda">
    <property name="title">
     <string>&amp;Ajuda</string>
    </property>
    <addaction name="action_Ajuda"/>
    <addaction name="separator"/>
    <addaction name="action_Sobre"/>
   </widget>
   <addaction name="menu_Ajuda"/>
  </widget>
  <action name="action_Ajuda">
   <property name="text">
    <string>&amp;Ajuda</string>
   </property>
  </action>
  <action name="action_Sobre">
   <property name="text">
    <string>&amp;Sobre</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
